03_API_Reference/01_Ethereum_Provider.md

```diff
@@ -92,11 +131,126 @@ params: [{
   "data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"
 }]
 
-ethereum.send({
+ethereum.sendAsync({
   method: 'eth_sendTransaction',
   params: params,
   from: accounts[0], // Provide the user's account to use.
+}, function (err, result) {
+  // A typical node-style, error-first callback.
+  // The result varies by method, per the JSON RPC API.
+  // For example, this method will return a transaction hash on success.
+})
+```
+
+### ethereum.autoRefreshOnNetworkChange (To Be Removed)
+
+_This will be removed on December 16, 2019. At this time, MetaMask will also stop reloading the page on network changes._
+_[Click here](https://medium.com/metamask/no-longer-reloading-pages-on-network-change-fbf041942b44) for more details._
+
+When the network is changed, MetaMask will reload any pages that have made requests to the provider.
+This automatic reload behavior will be removed in a future release of MetaMask, but in the meantime it can be disabled with this flag.
+
+To disable auto-refresh on a network change you can do:
+
+```javascript
+ethereum.autoRefreshOnNetworkChange = false;
+```
+
+This can be toggled on or off at any time.
+
+**Note:** Setting this flag to `true` results in the default behavior, which is subject to change. If your site relies upon MetaMask reloading it upon network change, you will need to trigger the reload yourself in a `networkChanged` event handler to ensure it continues to work with future releases.
+
+### ethereum.on(eventName, callback)
+
+The provider supports listening for some events:
+- `accountsChanged`, returns updated account array.
+- `networkChanged`, returns network ID string.
+
+#### Example
+```javascript
+ethereum.on('accountsChanged', function (accounts) {
+  // Time to reload your interface with accounts[0]!
+})
+```
+
+**Note:** At the moment, the `networkChanged` event is only useful if you disable auto-refresh on network
+change by setting [`ethereum.autoRefreshOnNetworkChange`](#ethereum.autorefreshonnetworkchange) to `false`.
+Otherwise, MetaMask will default to auto-reloading pages upon network change if they have made requests to the provider.
+_MetaMask will stop reloading pages on network change on December 16, 2019, and this setting will be removed._
+_[Click here](https://medium.com/metamask/no-longer-reloading-pages-on-network-change-fbf041942b44) for more details._
+
+# New API
+
+This API will become available during the week of **November 25, 2019**.
+It will be the only API supported starting **December 16, 2019.**
+If you want examples of how to setup your dapp using the new API, you can check out
+[this gist](https://gist.github.com/rekmarks/d318677c8fc89e5f7a2f526e00a0768a).
+
+## Methods
+
+### ethereum.send('eth_requestAccounts')
+
+#### ethereum.enable() (Deprecated)
```

Comment:
It wasn't immediately clear reading this that `ethereum.enable` was shown here because it's equivalent to `ethereum.send('eth_requestAccounts')`. Maybe referring to it as an alias/alternative/equivalent/synonym, or having a sentence to contextualize this would be helpful.